addons/event/models/event_event.py

```diff
@@ -644,6 +644,16 @@ def mail_attendees(self, template_id, force_send=False, filter_func=lambda self:
             for attendee in event.registration_ids.filtered(filter_func):
                 self.env['mail.template'].browse(template_id).send_mail(attendee.id, force_send=force_send)
 
+    def _get_external_description(self):
+        """ Description of the event shortened to maximum 2000 characters.
+        Meant to be used for external content (ics/icalc/Gcal).
+
+        Reference Docs for URl limit -: https://wiert.me/2010/04/20/maximum-url-lengths/
```

Comment:
Seems a bit outdated, but if we don't have better... I'll take it :smile:

Reply:
Hello @awa-odoo 
it is not documentation but I think it is answered by the Google AdWords API Team 
https://groups.google.com/g/adwords-api/c/1pUlvZQK58I/m/VPKLiZldAgAJ?pli=1

can we put this?  :smile:

Reply:
Let's put this one actually:
https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers

Even more direct.

(To adapt in the commit message as well).